metadata-ingestion/src/datahub/utilities/sqlglot_lineage.py

```diff
@@ -1024,6 +1024,18 @@ def _is_dialect_instance(
     return False
 
 
+def get_query_type(
```

Comment:
is this method still used anywhere? if not, can we remove it?

Reply:
removed